src/main/groovy/edu/oregonstate/mist/identify/IdentifyResource.groovy

```diff
@@ -0,0 +1,67 @@
+package edu.oregonstate.mist.identify
+
+import com.codahale.metrics.annotation.Timed
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.api.jsonapi.ResultObject
+import edu.oregonstate.mist.identify.db.IdentifyDAO
+import groovy.transform.TypeChecked
+import javax.annotation.security.PermitAll
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.Produces
+import javax.ws.rs.QueryParam
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+@Path("identify")
+@Produces(MediaType.APPLICATION_JSON)
+@PermitAll
+@TypeChecked
+class IdentifyResource extends Resource {
+    private IdentifyDAO identifyDAO
+    private URI endpointUri
+
+    IdentifyResource(IdentifyDAO identifyDAO, URI endpointUri) {
+        this.identifyDAO = identifyDAO
+        this.endpointUri = endpointUri
+    }
+
+    @Timed
+    @GET
+    Response identify(@QueryParam('osuID') String osuID,
+                      @QueryParam('facilityCode') String facilityCode,
+                      @QueryParam('cardID') String cardID) {
+        if (!osuID && !facilityCode && !cardID) {
+            return badRequest("No query parameters provided.").build()
```

Comment:
How about if we move these validation messages (the text) to the resource.properties file? We could then write tests for the validation logic